### PR TITLE
[PAY-157,PAY-160] Add TopSupporter/SupportingNotification

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Add TopSupporter/SupportingNotification
DuranDylan

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-157-160-top-supporters-notification

File: packages/web/src/components/notification/components/desktop/Notification.tsx [hunk 1]
```diff
@@ -67,9 +68,40 @@ const NotificationItem = (props: NotificationItemProps) => {
...
+    entities: props.entities,
+    value: 999,
+    reaction: 'fire'
   }
 
+  if (notification.type === NotificationType.RemixCosign) {
+    return <TipSentNotification notification={notification} />
+  }
+
+  // if (notification.type === NotificationType.Favorite) {
```
Comment (rickyrombo) [inline packages/web/src/components/notification/components/desktop/Notification.tsx:80]: probably need to clean up some test code/comments?

**Review by rickyrombo — CHANGES_REQUESTED** [review]

File: packages/web/src/components/notification/components/desktop/Notification.tsx (hunk 1 above)
Comment (dylanjeffers) [reply packages/web/src/components/notification/components/desktop/Notification.tsx:80]: oh yup good catch, this file wasn't supposed to be added!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-157-160-top-supporters-notification

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-157-160-top-supporters-notification

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-pay-157-160-top-supporters-notification

**Review by rickyrombo — APPROVED** [review]

**Pull request merged**